barretenberg/cpp/src/barretenberg/op_queue/ecc_ops_table.hpp

```diff
@@ -6,12 +6,29 @@
 #include <deque>
 namespace bb {
 
-enum EccOpCode { NULL_OP, ADD_ACCUM, MUL_ACCUM, EQUALITY };
+struct EccOpCode {
```

Comment:
Can you add some comments here explicitly detailing the formula here and explaining why its used?